apps/alphabet_mobile/yarn.lock

```diff
@@ -0,0 +1,6576 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
As above, we don't want a separate yarn.lock for this project, unless it ends up in a stand-alone repo.